templates/CRM/Member/Form/PaymentPlanToggler.tpl

```diff
@@ -104,12 +104,14 @@
 </script>
 <table id="payment_plan_fields">
   <tr id="contributionTypeToggle">
-    <td colspan="2" align="center">
-      <input name="contribution_type_toggle" id="contribution_toggle" value="contribution" type="radio">
-      <label for="contribution_toggle">Contribution</label>
-      &nbsp;
-      <input name="contribution_type_toggle" id="payment_plan_toggle" value="payment_plan" type="radio">
-      <label for="payment_plan_toggle">Payment Plan</label>
+    <td colspan="2">
+      <p>

```

Comment:
why do we need a `<p>` wrapping the input fields?

Reply:
To add the top-bottom padding to the toggle radio box set (that comes by default with p tag).